packages/core/src/components/Form.tsx

```diff
@@ -837,7 +837,7 @@ export default class Form<
     let field = this.formElement.current.elements[elementId];
     if (!field) {
       // if not an exact match, try finding an input starting with the element id (like radio buttons or checkboxes)
-      field = this.formElement.current.querySelector(`input[id^=${elementId}`);
+      field = this.formElement.current.querySelector(`input[id^="${elementId}"`);
```

Comment:
This seems like a straightforward change. Are you willing to add a test to verify it was fixed?

Reply:
@heath-freenome apologies, I made this PR right before going on vacation. I see it's been merged but would you still like me to add a unit test?